.github/workflows/benchmarks.yml

```diff
@@ -20,11 +34,13 @@ jobs:
       SSH_KEY: ./id_rsa_terraform
       TF_VAR_private_key: ./id_rsa_terraform
       TF_VAR_public_key: ./id_rsa_terraform.pub
-      TFVARS_SOURCE: 'system-profiles/8GBx1zone.tfvars' # // Default to use an 8gb profile
+      TFVARS_SOURCE: ${{ inputs.profile || 'system-profiles/8GBx1zone.tfvars' }} # // Default to use an 8gb profile
       TF_VAR_BUILD_ID: ${{ github.run_id }}
       TF_VAR_ENVIRONMENT: ci
       TF_VAR_REPO: ${{ github.repository }}
       GOBENCH_TAGS: branch=${{ github.head_ref || github.ref }},commit=${{ github.sha }},target_branch=${{ github.base_ref }}
+
+      BENCHMARK_AGENTS: ${{ inputs.benchmarkAgents || 64 }}
```

Comment:
I think having a default here will interfere with the autotuning in `make run-benchmark-autotuned`

Reply:
The benchmarks readme mentions 64 as being the default, so I assumed it would be fine. I've switched it to only be set if the value isn't empty.

Reply:
`BENCHMARK_AGENTS` defaults to 64 for `run-benchmark`, but we are using `run-benchmark-autotuned` which is essentially a wrapper over `run-benchmark` with a calculated `BENCHMARK_AGENTS`. See Makefile. Therefore `inputs.benchmarkAgents` will still be ignored in the current state of PR .

imo the right thing to do here to achieve what you want is that we `run-benchmark` instead of `run-benchmark-autotuned` when `inputs.benchmarkAgents` is set. Also good to have README updated to mention `run-benchmark-autotuned`

Reply:
I've changed the action, and documented the command.